### DISCUSSION TITLE
How to check whether my account has been upgraded to Team?

### DISCUSSION THREAD

**Discussion by ItaiBEdu** [community/community#65504] [GitHub Education] [answered]
I have a GitHub Education account as a verified teacher. One of the benefits I get is to upgrade my classrooms to GitHub Team. How can I check whether my classrooms have been upgraded to Team?

**Comment by kanazuchijin** [comment, marked as answer]
If you have already requested the upgrade and given approval, simply navigate to your organization and go to Settings --> Billing and Plans --> Current Plan. Instead of the version listed as Free you should see "GitHub Team." See the screen capture below.

![image](https://github.com/community/community/assets/57264954/06f14b50-693e-4df7-9d87-0202dba429b9)

**Reply by ItaiBEdu** [reply]
Thanks. I followed your instructions, and it says my plan is free. Perhaps I didn't request the upgrade properly. How can I request the upgrade?

**Reply by kanazuchijin** [reply]
You will need to navigate to this website, sign in to your account, and request the upgrade: [https://education.github.com/discount_requests/application](https://education.github.com/discount_requests/application). 

If you have not, you will need to verify your status as faculty.

**Comment by 0777505154a** [comment]
Ui mrc
